server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/template_config_representer.rb

```diff
@@ -0,0 +1,56 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Config
+    class TemplateConfigRepresenter < ApiV1::BaseRepresenter
+      alias_method :template, :represented
+
+      error_representer
+
+      link :self do |opts|
+        opts[:url_builder].apiv1_admin_template_url(template_name: template.name.to_s) unless template.name.nil? || template.name.to_s.blank?
```

Comment:
Can be simply written as `unless template.name.blank?` Since `CaseInsensitiveString#isBlank` exists and `nil` responds to `.blank?` :)

See http://api.rubyonrails.org/v4.0.4/classes/NilClass.html#method-i-blank-3F